File: LeetCode/2160. Minimum Sum of Four Digit Number After Splitting Digits/answer.py
# 2160. Minimum Sum of Four Digit Number After Splitting Digits
# https://leetcode.com/problems/minimum-sum-of-four-digit-number-after-splitting-digits/

class Solution:
    def minimumSum(self, num: int) -> int:
        sorted_num = sorted(str(num))

        result_1 = 0
        result_2 = 0
        for i in range(0, int(len(sorted_num)/2)):
            if i == 0:
                result_1 += int(sorted_num[i*2]) * 10
                result_2 += int(sorted_num[(i*2)+1]) * 10
            else:
                result_1 += int(sorted_num[i*2])
                result_2 += int(sorted_num[(i*2)+1])

        return result_1 + result_2

File: LeetCode/2620. Counter/answer.ts
// 2620. Counter
// https://leetcode-cn.com/problems/counter/

function createCounter(n: number): () => number {
  let num = n - 1;
  return function () {
    return (num += 1);
  };
}

/**
 * const counter = createCounter(10)
 * counter() // 10
 * counter() // 11
 * counter() // 12
 */
